docker-compose.yml

```diff
@@ -1,4 +1,39 @@
-version: "3"
+## HOW TO RUN: ##
+#
+# docker compose -f docker-compose.yml up -d
+#
+version: "3.4"
+
+x-g3w-suite-env: &g3w-suite-env
+  - FRONTEND
+  - G3WSUITE_GUNICORN_MAX_REQUESTS
+  - G3WSUITE_GUNICORN_NUM_WORKERS
+  - G3WSUITE_GUNICORN_TIMEOUT
```

Comment:
Check if the huey service requires setting a different / greater value for the timeout.